test/test_filters.rb

```diff
@@ -594,6 +603,19 @@ def to_liquid
         assert_equal "b", results[1]["id"]
         assert_equal "d", results[2]["id"]
       end
+
+      should "filter posts" do
+        site = fixture_site.tap(&:read)
+        posts = site.site_payload["site"]["posts"]
+        results = @filter.where_exp(posts, "obj", "obj.title == 'Foo Bar'")
+        assert_equal 1, results.length
+        assert_equal site.posts.find { |p| p.title == "Foo Bar" }, results.first
```

Comment:
`results.first` is a `Document`, not a `DocumentDrop`.
So... what should I do about that?